### ISSUE TITLE
instance Store s that fetches over the network

### ISSUE THREAD

**Issue by barischrooneyj** [barischrooneyj/simple-store#7]
An instance of the 'MonadStore' typeclass which fetches data over the network.

**Comment by barischrooneyj** [comment]
``` haskell
view a :: Storable a k => a -> m a
view a = read <$> get (concat ["/", typekey a, "/", key a])
```

**Comment by barischrooneyj** [comment]
Closed by 2ae00662a16c4b994a1f6535d28af65b74a07323
